src/background/ui.js

```diff
@@ -204,42 +207,10 @@ export class UI extends Component {
     }
 
     await Promise.all([
-      this.updateIcon(),
       this.sendDataToCurrentPort(),
     ]);
   }
```

Comment:
Not sure this needs `Promise.all()` anymore since it's just one function now. But removing it might also convert it from a Promise to something non-promisey. But then again the function seems to be `async` so probably a Promise regardless.

Reply:
oof. updated.